### ISSUE TITLE
What is still missing from the old league?

### ISSUE THREAD

**Issue by Dervishx5** [nbtdev/client#169]
Pirates
Hiring Hall
MRBC
Clan Honor
Black Market

**Comment by nbtdev** [comment]
We probably want to make an issue per item, that way we can discuss design/implementation independently for each

**Comment by Dervishx5** [comment]
I agree, however I'd like to use this for reference when I'm playing the Public Relations game. People are going to ask.
